packages/catalog/src/catalog-loader.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { ReactElement } from 'react';
+import { LoadingDots } from '@thoughtindustries/content';
+import { useCatalog } from './core';
+
```

Comment:
@ljiang-ti I think typing this component as follows would be better than as a JSX element since it is taking in props and returning a child component. What do you think?
```suggestion

interface CatalogLoaderProps {
  children: ReactElement;
}

```